po/es.po

```diff
@@ -11554,15 +11556,15 @@ msgstr "desplazamiento y"
 #. Let's add some tooltips and hook up some signals...
 #: ../src/iop/watermark.c:1479
 msgid "the opacity of the watermark"
-msgstr "opacidad de la marca de agua"
+msgstr "Transparencia de la marca de agua"
```

Comment:
A minor one, why not "Opacidad"?

I think in [photograpy terms we're looking for opacity](https://helpx.adobe.com/es/photoshop-elements/key-concepts/transparency-opacity.html).

Reply:
Ese cambio lo hice por una razón que ahorita no tengo a la mano. Jajaja, pero lo reviso.